clang/test/SemaCXX/complex-div.cpp

```diff
@@ -0,0 +1,3 @@
+// RUN: %clang_cc1 -fsyntax-only %s
+
+auto f() { return 43273096 / 65536j; }
```

Comment:
can you move that to , for example `clang/test/SemaCXX/constant-expression-cxx11.cpp`
and make sure that actually always get evaluated?
maybe `constexpr auto a = `43273096 / 65536j;`